src/index.js

```diff
@@ -4,15 +4,34 @@ import './style.css';
 import { Operations } from './modules/operations.js';
 const taskContainer = document.getElementById('task-container');
 const submitButton = document.getElementById('addButton');
-// const clearAllDone = document.getElementById('clearComplete');
+const clearAllDone = document.getElementById('clearComplete');
 let editButtonStats=false;
 window.onload = function windowReady() {
   Operations.showTask();
   submitButton.onclick = function () {
     Operations.createTask();
   };
   
-
+  taskContainer.addEventListener('click', (e) => {
+    if (e.target !== null && e.target !== 'NaN' && e.target !== '') {
+      if (e.target.className === 'checkbox-class') {
+        const ids = e.target.id.replace('checkbox-', '');
+        const description = document.getElementById('d' + ids);
+        const data = Operations.getAllTasks();
+        const index = parseInt(ids-1, 10);
+        if (data !== []) {
+          if (data[index].completed) {
+            data[index].completed = false;
+            description.style.textDecoration = 'none';
+          } else {
+            data[index].completed = true;
+            description.style.textDecoration = 'line-through';
+          }
+          Operations.updateTask(data);
+        }
+      }
+    }
+  });
```

Comment:
- [ ] Great job so far. There's little issue here, when you mark some tasks completed and click on the `Clear all completed` button to delete all the tasks marked as completed, the tasks get deleted but the indexes of the remaining tasks are not re-ordered so when a new task is added, there's every possibility that two tasks would have the same index number which will make it difficult to mark them completed and save them in the local storage. This is what you need to work on before your project could be approved. Kindly fix that. 

[app-behave.webm](https://user-images.githubusercontent.com/26597432/229086380-a3092c10-a362-4824-a6f2-8130e4ebdf5b.webm)

Reply:
resolved as well